package.json

```diff
@@ -54,7 +54,12 @@
     "tsc": "tsc",
     "lint": "npm-run-all -p lint:*",
     "lint:ts": "tslint -p . --format verbose",
-    "lint:sass": "sass-lint -v"
+    "lint:sass": "sass-lint -v",
+    "start:examples": "npm run start -- --examples",
+    "e2e": "start-server-and-test start:examples http://localhost:9090/health/ready cy:run",
+    "e2e:dev": "start-server-and-test start:examples http://localhost:9090/health/ready cy:open",
```

Comment:
`e2e:watch`?

Reply:
It runs different mode of cypress and I like "dev" suffix.